doc/mediasdk-man.md

```diff
@@ -9255,6 +9263,110 @@ The `SegmentFeature` enumerator indicates features enabled for the segment. Thes
+
+**Name/Description**
+
+| | |
+--- | ---
+`MFX_AV1_SEGMENT_DISABLED` | No segmentation in coded stream
+`MFX_AV1_SEGMENT_AUTO`     | Segmentation map and segment features/values are set by the SDK based on analysis of a frame
+`MFX_AV1_SEGMENT_MANUAL`   | Segmentation map and segment features/values are provided by application using [mfxExtAV1Segmentation](#mfxExtAV1Segmentation) structure;
+
+**Change History**
+
+This enumerator is available since SDK API **TBD**.
+
+## <a id='AV1SegmentIdBlockSize'>AV1SegmentIdBlockSize</a>
+
+**Description**
+
+The `AV1SegmentIdBlockSize` enumerator indicates the block size represented by each segment_id in segmentation map for AV1 encoder. These values are used with the [mfxExtAV1Segmentation](#mfxExtAV1Segmentation)**::SegmentIdBlockSize** parameter.
+
+**Name/Description**
+
+| | |
+--- | ---
+`MFX_AV1_SEGMENT_ID_BLOCK_SIZE_UNKNOWN` | Unspecified block size
+`MFX_AV1_SEGMENT_ID_BLOCK_SIZE_8x8`     | 8x8 block size
+`MFX_AV1_SEGMENT_ID_BLOCK_SIZE_16x16`   | 16x16 block size
+`MFX_AV1_SEGMENT_ID_BLOCK_SIZE_32x32`   | 32x32 block size
+`MFX_AV1_SEGMENT_ID_BLOCK_SIZE_64x64`   | 64x64 block size
+
+**Change History**
+
+This enumerator is available since SDK API **TBD**.
+
+## <a id='AV1SegmentFeature'>AV1SegmentFeature</a>
+
+**Description**
+
+The `AV1SegmentFeature` enumerator indicates features enabled for the segment for AV1 encoder. These values are used with the [mfxAV1SegmentParam](#mfxAV1SegmentParam)**::FeatureEnabled** parameter.
+
+**Name/Description**
+
+| | |
+--- | ---
+`MFX_AV1_SEGMENT_FEATURE_ALT_QINDEX`      | Quantization index delta
+`MFX_AV1_SEGMENT_FEATURE_ALT_LF_Y_VERT`   | Loop filter level delta for Y component vertical direction
+`MFX_AV1_SEGMENT_FEATURE_ALT_LF_Y_HORZ`   | Loop filter level delta for Y component horizontal direction
+`MFX_AV1_SEGMENT_FEATURE_ALT_LF_U`        | Loop filter level delta for U component
+`MFX_AV1_SEGMENT_FEATURE_ALT_LF_V`        | Loop filter level delta for V component
+`MFX_AV1_SEGMENT_FEATURE_REFERENCE`       | Reference frame
+`MFX_AV1_SEGMENT_FEATURE_SKIP`            | Skip
+`MFX_AV1_SEGMENT_FEATURE_GLOBALMV`        | Global Motion
+
+**Change History**
+
+This enumerator is available since SDK API **TBD**.
+
+## <a id='AV1ReferenceFrame'>AV1ReferenceFrame</a>
+
+**Description**
+
+The `AV1ReferenceFrame` enumerator itemizes reference frame type specified by [mfxAV1SegmentParam](#mfxAV1SegmentParam)**::ReferenceFrame** parameter.
+
+**Name/Description**
+
+| | |
+--- | ---
+`MFX_AV1_REF_INTRA`   | Intra
+`MFX_AV1_REF_LAST`    | Last
+`MFX_AV1_REF_LAST2`   | Last 2
+`MFX_AV1_REF_LAST3`   | Last 3
+`MFX_AV1_REF_GOLDEN`  | Gold
+`MFX_AV1_BWDREF`      | Backward
+`MFX_AV1_REF_ALTREF`  | Alternative
+`MFX_AV1_REF_ALTREF2` | Alternative 2
+
+**Change History**
+
+This enumerator is available since SDK API **TBD**.
 
 ## <a id='InsertHDRPayload'>InsertHDRPayload</a>
```

Comment:
@wangyan42164 , it seems to be encoder related. Please remove it for now.

Reply:
Done. Sorry for this.